working/macros/example/bin/checks_main.dart

```diff
@@ -0,0 +1,23 @@
+@ChecksExtensions([Person])
+library;
+
+import 'package:checks/checks.dart';
+import 'package:test/test.dart';
+
+import 'package:macro_proposal/checks_extensions.dart';
+
+void main() {
+  test('can use generated extensions', () {
+    final draco = Person(name: 'Draco', age: 39);
+    check(draco)
+      ..name.equals('Draco')
+      ..age.equals(39);
+  });
+}
+
+class Person {
```

Comment:
Can we make an example where the class we generate for is imported?

Reply:
It doesn't make any difference in this case - and the boilerplate required right now for running these is already a lot, I don't really want to add more files.

For the real macro, once you can run it in a more sane way, you could absolutely do that.